tidy3d/components/data/monitor_data.py

```diff
@@ -1989,16 +1990,17 @@ class DirectivityData(MonitorData):
 
     Example
     -------
-    >>> from tidy3d import DirectivityDataArray
+    >>> from tidy3d import DirectivityDataArray, AxialRatioDataArray
     >>> f = np.linspace(1e14, 2e14, 10)
     >>> r = np.atleast_1d(5)
```

Comment:
This is pedantic, but the radiation characteristics should be independent of r, since radiation intensity is scaled by r^2. Including r is not required, maybe if we rename later we can remove r, unless I am missing something.

Reply:
Actually this projection distance r is used to compute the propagation factor to get the correct Etheta and Ephi.

Reply:
That's right, but it ultimately isn't necessary for describing the resulting Directivity. Directivity only depends on frequency and angular position, the same as radiation intensity.

Reply:
That makes sense.

Reply:
I just found that if we give up introducing the coordinate "r" here, we will need to do a corresponding coordinate change in the backend, which will make the code ugly. As this class inherits from FieldProjectionAngleMonitor, which has the coordinate "r", it's better to keep it consistent I guess?  Users typically don't need to specify this value and can use the default value of 1e6.

Reply:
Yea, totally fine to keep it. I just wanted to point out that we don't absolutely need it in these DataArrays.